src/app/payment/payment-content.tsx

```
@@ -17,61 +32,81 @@ import { maskEmail, maskPhoneNumber } from "@/lib/utils/mask-data";
-          selected ? 'border-green-500 bg-green-50' : ''
-        }`}
+    <motion.div
+      whileHover={{ scale: 1.02 }}
+      whileTap={{ scale: 0.98 }}
+      className="w-full"
+    >
+      <div
+        className={`p-4 cursor-pointer transition-all duration-300 bg-white rounded-xl border ${
+          selected ? "border-green-500 bg-green-50/50" : "border-gray-100"
+        } hover:shadow-md`}
+        role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)